package.json

```diff
@@ -63,7 +63,7 @@
     "estraverse": "4.2.0",
     "extend": "3.0.2",
     "gettext-parser": "1.3.0",
-    "grunt": "1.0.3",
+    "grunt": "^1.0.4",
```

Comment:
AFAIK we don't want to use `^`, we want to use exact version

Reply:
sure 👍